ssg/build_remediations.py

```diff
@@ -691,24 +738,27 @@ def expand_xccdf_subs(fix, remediation_type, remediation_functions):
         patcomp = re.compile(pattern, re.DOTALL)
         fixparts = re.split(patcomp, fix.text)
         if fixparts[0] is not None:
```

Comment:
I can't wait for the moment when we take some of the existing magic out and simplify this function.

Reply:
I suspect that when all bash remediation functions are converted to Jinja macros `expand_xccdf_subs` won't be needed anymore.